docs/hugo/content/introduction/annotations.md

```diff
@@ -0,0 +1,27 @@
+# Annotations understood by the operator
+
+## Annotations specified by the user
+
+Note that unless otherwise specified, allowed values are _case sensitive_ and should be provided in lower case.
+
+1. `serviceoperator.azure.com/reconcile-policy`: Specifies the reconcile policy to use. Allowed values are:
+    - `run`: The operator performs all actions as normal. This is the default if no annotation is specified.
+    - `skip`: All modification actions on the backing Azure resource are skipped. GETs are still issued to ensure that the resource
+    exists. If the resource doesn't exist, the `Ready` condition will show a `Warning` state with the details of the missing resource
+    until the resource is created. If the resource is deleted in Kubernetes, it is _not_ deleted in Azure. In REST API terminology, 
+    PUT and DELETE are skipped while GET is allowed.
+    - `skip-delete`: Modifications are pushed to the backing Azure resource, but if the resource is deleted in Kubernetes, 
```

Comment:
@theunrepentantgeek brought up a concern regarding the "delete-by-default even for resources we may not have created" which this implies. He asked if there was a way to know if we created the resource and base the default behavior on that instead. So for resources which ASO created it would default to deleting them but for resources it adopted it would default to abandoning them instead (not issuing a DELETE to Azure).

Just making this comment here as a place for discussion, I'll be doing some thinking about this and reply later.

Reply:
I'm thinking there's a particular risk here for stateful resources that can't simply be recreated from their spec - the most obvious of these being various kinds of database.

Reply:
I've written a little design up [here](https://github.com/Azure/azure-service-operator/pull/2067). I think it makes sense to decouple that from this PR though, as it's really two things:

1. Support for `reconcile-policy` at all.
2. The default behavior of `reconcile-policy` for adopted resources (as well as the ability to actually detect if we _are_ adopting a resource, which will take additional controller changes).

Once 1 exists we can do 2 (if we decide that we want to).

Reply:
Resolving this: Let's move conversation on this topic to the design around it [here](https://github.com/Azure/azure-service-operator/pull/2067).